### ISSUE TITLE
Mac performance is poor 

### ISSUE THREAD

**Issue by 9SL9** [skiselkov/librain#12]
The plugin works on Mac but the FPS hit is huge

**Comment by KarlLeclerc** [comment]
Agreed. I have tested in the last 2 versions of MacOS, and on different aircrafts. The loss of FPS averaged 80 to 90 %. FPS would drop from 40 to 4 for example. While it may work as in "it does not crash", it is not useable on Mac's.
